connector/src/models/Honeypot.ts

```diff
@@ -0,0 +1,82 @@
+import Config from './Config'
```

Comment:
I think for internal naming it's fine to have it named as is, but let's keep it in mind for the future yes.
Not sure what did you mean on changing Honeypot references to ConvictionVoting? Since Honeypot does not only refer to conviction voting.